packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -443,19 +443,22 @@
 </section>
 
 <section>
-  <h3 class="dummy-h3">🚧 Accessibility: Alert (Page), Alert (Inline), and Alert (Compact)</h3>
-  <p class="dummy-paragraph">
-    The Alert (Page) has
-    <code class="dummy-code">role="alert"</code>
-    on the parent
-    <code class="dummy-code">div</code>, to identify the element as the container where alert content will be added or
-    updated.
-  </p>
+  <h3 class="dummy-h3">🚧 Accessibility</h3>
```

Comment:
Wanted to specifically call out this section, from the preview link https://hds-components-git-amy-alert-a11y-hashicorp.vercel.app/components/alert:

<img width="1116" alt="image" src="https://user-images.githubusercontent.com/1372946/161896850-ae70a665-bd03-458b-b743-f5839527242f.png">

LMK what you think, needs some more verbiage and logic in code @didoo @cveigt @MelSumner

Reply:
> When the dismiss button is focused, it should close on return / enter
 should it be `return / enter` or `space`?

I've seen that some design systems also use role=status depending on the alert type. I'm going to read a bit more tomorrow. [IBM's Carbon](https://www.carbondesignsystem.com/components/notification/accessibility) is one of them.

Also, this section does not include AlertToast, right?

Reply:
@cveigt Let me check on `space`. Where did you see that? I've been doing a lot of keyboard tabbing cause the spec doesn't always list (each one seems pretty scoped). I *think* Enter is for PC.

Am suggesting role="status" for neutral, highlight, and success

For Toast...it's sort of tricky. I think it's arguable that all Toasts could be an alertdialog. Seems like most of the ones we have, have actions => alertdialog. But I think the type is the determinant

Also that reminds me, should check on the Toast dismiss, if it's ESC.

Reply:
@cveigt Just met with Cristiano and Brian on an eng handoff plan for before April 16. Since we are using the button you are totally right about space. (And we can stick with hover, focus, active to match the button, which is what's on the preview link rn,)

I'm going to flesh out the a11y more, but will do some testing in cloud-ui first. I can still update this by EOD and then will be out Thurs & Fri, here all next week before transferring.

There may be some Toast-specific a11y recommendations (how long before it dismisses, details about the animation), but we won't provide the flash service AFAICT. Specifically will work on a 1:1 swapout of https://cloud-ui-storybook-hashicorp.vercel.app/?path=/story/notifications--notifications for Alert (Toast). Also we can talk at standup

Reply:
updated ^, resolving